ads/nativo.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../3p/3p';
+
+export function nativo(global, data) {
+
+  let ntvAd;
+  (function(ntvAd, global, data) {
+    global
+      .frames
+      .location
+      .hash = global.frames.location.hash.replace(/({).*(})/,'');
```

Comment:
The way our code works it needs the url. Your AMP code throws unnecessary JSON into the URL, which breaks our on our server side. If you have another suggestion then please advise.